src/xunit.performance.api/Profilers/Etw/EtwManagedMethod.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Xunit.Performance.Api.Profilers.Etw
+{
+    internal sealed class EtwManagedMethod
```

Comment:
I would recommend not using the Etw prefix on these classes.  You could imagine getting this data from other profilers, and we'd like to be able to do so, especially on non-windows.

Reply:
Makes sense, but I am not sure, at this point, if the internal data/types are applicable to other profilers. I've been looking into other ones, and once I get started it will be more clear what direction to take.
I am thinking using interfaces instead. What do you think?

---
In reply to: [143033861](https://github.com/Microsoft/xunit-performance/pull/241#discussion_r143033861) [](ancestors = 143033861)

Reply:
Yup, interfaces seem reasonable.

---
In reply to: [143237043](https://github.com/Microsoft/xunit-performance/pull/241#discussion_r143237043) [](ancestors = 143237043,143033861)